### PR TITLE
(CDAP-980) Added sendfile command to CLI to send file content to a stream

### PR DESCRIPTION
**Pull request by chtyim**

### CODE REVIEW

File: cdap-cli/src/main/java/co/cask/cdap/cli/command/SendFileStreamCommand.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    File file = new File(arguments.get(ArgumentName.LOCAL_FILE_PATH.toString()));
+    String contentType = arguments.get(ArgumentName.CONTENT_TYPE.toString(), "");
+
+    if (contentType.isEmpty()) {
+      contentType = getContentType(Files.getFileExtension(file.getName()));
+    }
+    if (contentType.isEmpty()) {
+      throw new IllegalArgumentException("Unsupported file format.");
+    }
+    if (!file.isFile()) {
```
Comment (jwang47) [inline cdap-cli/src/main/java/co/cask/cdap/cli/command/SendFileStreamCommand.java]: This check should be done before the content type check I think.

Comment (chtyim) [inline cdap-cli/src/main/java/co/cask/cdap/cli/command/SendFileStreamCommand.java]: Either way is fine as `getName` doesn't require file to exists. Anyway, moving it up.

**Comment by jwang47** [comment]
Code looks good, but the command format for sending things to stream is getting confusing.

```
send stream <..> <text>: Sends <text> to the stream
sendfile stream <file>: Sends contents of <file> to the stream
```

How about something like

```
send stream <stream-id> text <text>
send stream <stream-id> file <file> content-type <content-type>
```

**Comment by chtyim** [comment]
why send/sendfile is confusing?

**Comment by jwang47** [comment]
"send" doesn't specifically say what you can send, yet you can only send text. Even sendtext/sendfile would be better, but I like having send and text/file as separate words. I decided I liked having "send stream <> text" rather than "send text stream <>" since the command format for the other commands are mostly in general -> specific in terms of the keyword order.

**Comment by chtyim** [comment]
But sendfile is very explicit that it's sending file (which is a batch request), while send is sending one by one. Making them as "send" but sometime is one event, sometime is a batch is more confusing to me.

**Comment by jwang47** [comment]
I see what you mean now. How about this?

```
send stream <stream-id> batch <file> content-type <content-type>: Sends batch file which may be treated as multiple stream events
send stream <stream-id> file <file>: Sends the contents of a file to be treated as a single stream event 
send stream <stream-id> text <text>: Sends text to be treated as a single stream event
```

or maybe

```
send stream <stream-id> batch-file <file> content-type <content-type>: Sends batch file which may be treated as multiple stream events
```

**Comment by chtyim** [comment]
Isn't that too much to type? Besides, from the CLI, the only way to send batch is through a file. We could do some nasty multi-line input with Ctrl-D as EOF. But seems defeat the purpose of CLI.

**Comment by anew** [comment]
My 2cent:
send stream <stream name> text "text of the event" [ headers a=1,b=2 ]
send stream <stream name> file <file name>
load stream <stream name> <file name> [ <content type> ]

**Comment by anew** [comment]
For this release, we only need load.

**Comment by jwang47** [comment]
LGTM

**Pull request merged**